### PR TITLE
Add some more reward constants

### PR DESCRIPTION
**Pull request by Stebalien**
TODO: Replace THING with what the thing actually is.

Actually, it would be really nice to have descriptions of all of these things.

----

#

### CODE REVIEW

File: actors/builtin/reward/reward_calc.py [hunk 1]
```diff
@@ -15,6 +15,9 @@
 # Precision factor.
 Q128 = 2**128
 
+# TODO: what is this?
+THING=330000000
```
Comment (Stebalien) [inline actors/builtin/reward/reward_calc.py:19]: Ah, is this the supply total?

Comment (anorth) [reply actors/builtin/reward/reward_calc.py:19]: The simple supply total

File: actors/builtin/reward/reward_calc.py [hunk 2]
```diff
@@ -26,6 +30,9 @@ def epochs_in_year() -> Decimal:
 def q128(val, round) -> str:
     return str((Q128 * val).to_integral_value(round))
 
+def atto(val) -> str:
+    return str((10**18 * val).to_integral_value(decimal.ROUND_DOWN))
```
Comment (Stebalien) [inline actors/builtin/reward/reward_calc.py:34]: @davidad I'm not finding any rhyme or reason to our rounding. Can we just safely round down (i.e., truncate)?

Comment (davidad) [reply actors/builtin/reward/reward_calc.py:34]: Fine with me. All our fixed-point arithmetic does truncation too, we just aim to have enough precision that it doesn't matter.

**Pull request merged**